src/index.js

```diff
@@ -0,0 +1,26 @@
+import config from 'config';
+import express from 'express';
+
+import apiRouter from './routes/index.js';
+
+import errorsMiddleware from './middlewares/errors.js';
+import loggerMiddleware from './middlewares/logger.js';
+
+const app = express();
+
+if (process.env.NODE_ENV !== 'test') {
+  app.use(loggerMiddleware);
+}
+
+export const basePath = '/v1';
```

Comment:
Since this is a const, shouldn't it be:
```suggestion
export const BASE_PATH = '/v1';
```
?